### PR TITLE
Ft better ui and logic 153324164

### PR DESCRIPTION
**Pull request by Kalela**
#### What does this PR do?
Switches the project folder structure

#### How should this be manually tested?
Tested by pushing local project to github. Travis CI checks provided tests if Build is working.
 
#### What are the relevant pivotal tracker stories?
Create tests for upcoming functionality
ID: #153294351

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+source venv/bin/activate
```
Comment (Ashaba) [inline .env]: why would you want to activate your virtualenv from the .env file?

File: README.md [hunk 2]
```diff
@@ -1,74 +1,65 @@
 Event Hub
 =========
 Event Hub is an offshoot website of of Bright Events collaborative development training, hosted by Andela. The Site allows users to view and manage their own events as well as inform potential attendants of the event's existence. Visit our link at https://kalela.github.io/BrightEvents/.
```
Comment (Ashaba) [inline README.md]: I would like you take recommendations raised in the issue to make your readme more informative.

File: designs/UI/delete_event.html [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-</body>
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Delete Event</title>
+</head>
+<body>
+
+</body>
 </html>
```
Comment (Ashaba) [inline designs/UI/delete_event.html]: CRITICAL

expected end of file line

File: tests.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        self.app = create_app(config_name="testing")
+        self.app.config['SECRET_KEY'] = 'my-secret-key'
+        with self.app.app_context():
+            pass
+
+    def test_register_json(self):
+        tester = self.app.test_client(self)
+        response = tester.post('/api/v2/auth/register',
+                               data=dict(username = "admin", password = "1234"))
+        self.assertEqual(response.status_code, 201, msg="Register api not working")
```
Comment (Ashaba) [inline tests.py]: your status code says created, the message indicates the api not working, could you kindly resolve this

File: tests.py [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        self.assertIn("1234", str(response.data))
+
+    def test_login_json(self):
+        tester = self.app.test_client(self)
+        response = tester.post('/api/v2/auth/login',
+                               data=dict(username = "user", password = "password"))
+        self.assertEqual(response.status_code, 201, msg="Login api not working")
```
Comment (Ashaba) [inline tests.py]: here as well, check on the status code

File: tests.py [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+        tester = self.app.test_client(self)
+        response = tester.get('/api/v2/events')
+        self.assertEqual(response.status_code, 200, msg="View all events api not working")
+        self.assertIn("user events", str(response.data))
+
+    def test_send_rsvp_json(self):
+        tester = self.app.test_client(self)
+        tester.post('/api/v2/auth/login', data=dict(username = "user", password = "password"))
+        response = tester.post('/api/v2/events/MyParty/rsvp')
+        self.assertEqual(response.status_code, 201, msg="Send rsvp api not working")
```
Comment (Ashaba) [inline tests.py]: Ensure you are testing the expected working/functionality of your code.

File: .env (hunk 1 above)
Comment (Kalela) [reply .env]: So that autoenv kicks in and activates the virtualenv automatically every time i cd into my project

File: tests.py (hunk 4 above)
Comment (Kalela) [reply tests.py]: Okay.

File: .env (hunk 1 above)
Comment (ebrahim-j) [reply .env]: If it works using autoenv, then that's good stuff

**Comment by ebrahim-j** [comment]
Haven't gone through your code but I'm not sure why we have so much being done on this branch..
Ideally, new code added/ edited/ removed must directly be linked to what the branch name is, so unless all these commits are UI and logic **only**, then it's too much being done.
Also the branch name would probably need to be less vague(for the next branches) and have some sort of indication of what specifically is being worked on eg `ft_user_auth_logic` rather than just `ft_logic` if you're working on user authorization logic..
Lastly, I would recommend that PR's have just a handful of files being changed as it would be easier for reviewing purposes

**Comment by Kalela** [comment]
Got it. I'll make branches less ambiguous from now on.

**Pull request merged**